tests/console/firewalld.pm

```diff
@@ -8,7 +8,7 @@
 # without any warranty.
 
 # Package: firewalld
-# Summary: Test FirewallD basic usage
+# Summary: Test FirewallD basic usage, including NFT tables
```

Comment:
LCD display!

```suggestion
# Summary: Test FirewallD basic usage, including nftables/iptables
```

Reply:
https://knownorigin.io/gallery/254225-i-don-t-know-look-at-it-ix!